ha/util/conf_store.py

```diff
@@ -221,3 +221,30 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_pods(index):
```

Comment:
Please rename to get_control_nodes

Reply:
ACK

Reply:
As suggested by @ujjwalpl,
need to change other func names as well
get_server_pods -> get_server_nodes
get_data_pods -> get_data_nodes

This can be take up as a seperate task as an code improvemet ticket.
@shaileshsvc @ArchanaLimaye

Reply:
@gargpalak048  PLease open a Jira ticket (Bug) for this and post the Jira ID here so that this comment can be marked resolved.

Reply:
Ticket created: https://jts.seagate.com/browse/CORTX-30767 
This will be improvement story ticket, not a Bug

Reply:
Resolving the comment based on discussion with Ujjawal.